app/models/agents/aftership_agent.rb

```diff
@@ -0,0 +1,158 @@
+require 'uri'
+
+module Agents
+  class AftershipAgent < Agent
+
+    default_schedule "every_10m"
+
+    description <<-MD
+      The Aftership agent allows you to track your shipment from aftership and emit them into events.
+
+      To be able to use the Aftership API, you need to generate an `API Key`. You need a paying plan to use their tracking feature.
+
+      You can use this agent to retrieve tracking data. You have to provide a specific `get` request and its associated option.
+ 
+      To get all trackings for your packages please enter `get` for key and `trackings` for the option.
+      To get tracking for a specific tracking number, add the extra keys `slug`, `tracking_number` and its associated values.
+
+      Set `single_tracking_request` to true as well.
+
+      To get the last checkpoint of a package set key to `get` and option to `last_checkpoint`. Please provide `slug` and `tracking_number`
+
+      `slug` is a unique courier code. 
+
+      You can get a list of courier information here `https://www.aftership.com/courier`
+
+      Required Options:
+
+      * `api_key` - YOUR_API_KEY.
+      * `get and its associated options`
+    MD
+
+    event_description <<-MD
+      A typical tracking event has 3 objects (attributes, tracking, and checkpoint) and it looks like this
+          {
+        "meta": {
```

Comment:
The indentation looks a bit off here, I think the complete hash needs to be indented with 4 spaces so markdown renders it as a block like so?
<img width="541" alt="screenshot 2016-03-16 09 36 39" src="https://cloud.githubusercontent.com/assets/20943/13806269/dfde70d0-eb5a-11e5-90a3-e63596e9aa89.png">